Making updates for 2.7

dockerymick opened the pull request:
https://github.com/stolostron/backlog/issues/25892

dockerymick committed:
Making updates for 2.7

birsanv commented on backup_restore/manage_backup_restore.adoc:
```diff
@@ -53,6 +53,7 @@ Complete the following steps to create an instance of the `DataProtectionApplica
 . From the {ocp} console, select *Operators* > *Installed Operators*.
 . Click `Create instance` under DataProtectionApplication.
 . Create the Velero instance by selecting configurations using the {ocp-short) console or by using a YAML file as mentioned in the `DataProtectionApplication` example.
+. Set the `DataProtectionApplication` namespace to `<acm-namespace>-backup`.
```
I just found out, the ns is **always**  open-cluster-management-backup so we can call it directly : 
Set the `DataProtectionApplication` namespace to `open-cluster-management-backup`.

dockerymick committed:
Update manage_backup_restore.adoc

Update after review from Valentina

dockerymick committed:
Update manage_backup_restore.adoc

removing note to reflect similar changes from the [PR (2.6 changes)](https://github.com/stolostron/rhacm-docs/pull/4231)

dockerymick commented:
Applied the necessary changes. Comfortable with merging this PR based on my previous [PR](https://github.com/stolostron/rhacm-docs/pull/4231)
